apps/examples/testcase/ta_tc/mqtt/itc/itc_mqtt_main.c

```diff
@@ -65,13 +60,8 @@ static int _itc_mqtt_init(void)
 {
 	g_mqtt_client_handle = NULL;
 
-	int res = pthread_mutex_init(&g_ocfmutex, NULL);
-	if (res != 0) {
-		ITC_MQTT_LOGE;
-		return -1;
-	}
-	res = pthread_cond_init(&g_ocfcond, NULL);
-	if (res != 0) {
+	int ret = sem_init(&g_mqtt_signal, 0, 0);
```

Comment:
Please confirm whether mqtt utc uses the same variable. If so, maybe rename this variable.

Reply:
review comment resolved